src/main/java/core/basesyntax/strategy/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public int doCalculation(int totalQuantity, int quantity) {
+        return totalQuantity += quantity;

```

Comment:
- This method should return nothing and should store the result of the operation to the `Storage` map.
- Please fix this suggestion in all  `*Handler` classes.
- Do not forget check the balance (e.g. negative balance) when operation processed and throw the runtime exceptions inside of `doCalculation()` methods